mapstruct/src/test/java/com/baeldung/expression/mapper/LicenseMapperUnitTest.java

```diff
@@ -0,0 +1,97 @@
+package com.baeldung.expression.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+
+import com.baeldung.expression.dto.LicenseDto;
+import com.baeldung.expression.model.License;
+
+class LicenseMapperUnitTest {
+
+    LicenseMapper licenseMapper = Mappers.getMapper(LicenseMapper.class);
+
+    @Test
+    void givenLicenseDtoWithStartDateAndWithoutEndDate_WhenMapperMethodIsInvoked_ThenLicenseShouldBePopulatedWithDefaultEndDate() {
+        License license = licenseMapper.toLicense(LicenseDto.builder()
+            .startDate(LocalDateTime.now())
+            .build());
+        assertThat(license).isNotNull()
+            .satisfies(l -> {
```

Comment:
Instead of satifies, simply perform the assertions line by line, I think it is easier to read that way - unless there is some other reason for using satifies? Similar in the other tests too

e.g. 
```
assertThat(license).isNotNull();
assertThat(license.getStartDate().toLocalDate()).isEqualTo(LocalDate.now());
```